### ISSUE TITLE
Accessibility Request

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#1009]
I've received a request from, I guess, a blind individual who'd like to play RIS but of course can't see it.  This is all way over my head, so I'll paste his request in here.  I have no idea what would be involved with such a change, but here's his message:

Hi! this game sounds very interesting, unfortunately I wasn't able to play it because it doesn't send any text to my screenreading software. Is that something that could be feasibly added? Windows has a library called tolk which is in a number of projects and seems to do a good job of this. I don't know Linux very well but suspect sending the text to speech dispatcher would be enough. OSX is out of my wheelhouse unfortunately.

I can't see the screen, so I need the screenreader to do things on the computer. It does seem to have quite a few keyboard shortcuts which is great, but when I review the window the screenreader only reads the window title, so I can't read the text in the game. The help/keyboard instructions, the option I'm currently focused on, text associated with my chosen space hardware. And so on. I believe the culprit here is SDL: https://github.com/libsdl-org/SDL/issues/7463

I don't mean to propose a major change to the game itself, you'd certainly know better than me about how it's structured, but sending the text to accessibility APIs in addition to graphical ones should be possible. There are a few workarounds mentioned in that thread to speed up the process, though windows appears to have the most mature support in this area currently.

[https://mstdn.games/@prism@infosec.exchange/115347521800971649](url)

**Comment by peyre** [comment]
Good point.  I'll tag this as Epic.
